test/io/client_test.dart

```diff
@@ -9,6 +9,7 @@ import 'dart:io';
 
 import 'package:http/http.dart' as http;
 import 'package:http/src/io_client.dart' as http_io;
+import 'package:http/src/io_streamed_response.dart' as http_io_response;
```

Comment:
I think if we import `package:http/io_client` instead then we can pick up both types we need in one place, and it would better represent real uses of this package.
```suggestion
import 'package:http/io_client.dart' as http_io;
```

Reply:
👍 Updated in b699b87.